packages/blockchain/ada/jest.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  displayName: 'blockchain-ada',
+  preset: '../../../jest.preset.js',
+  globals: {
+    'ts-jest': {
+      tsconfig: '<rootDir>/tsconfig.spec.json',
+    },
+  },
+  transform: {
+    '^.+\\.[tj]s$': 'ts-jest',
+  },
+  moduleFileExtensions: ['ts', 'js', 'html'],
```

Comment:
I think it should be 

`moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json'],`